test/pa_ess/utilities_test.exs

```diff
@@ -109,4 +109,11 @@ defmodule Content.Audio.UtilitiesTest do
       assert replace_abbreviations("SvC, OK") == "Service, OK"
     end
   end
+
+  test "paginate_text" do
+    assert [{"Attention passengers:", "the next Braintree train", 3}, {"is now arriving", "", 3}] =
+             paginate_text("Attention passengers: the next Braintree train is now arriving", 24)
+
+    assert [{"too-long", "word", 3}] = paginate_text(" too-long   word ", 5)
+  end
```

Comment:
**suggestion:** can we add a test where the message fits within the `max_length`?